src/metabase/api/common.clj

```diff
@@ -26,6 +26,10 @@
    ex. `@*current-user*`"
   (atom nil)) ; default binding is just something that will return nil when dereferenced
 
+(def ^:dynamic ^Boolean *is-superuser?*
+  "Is the current user a superuser?"
+  false)
+
```

Comment:
@salsakran variables with earmuffs around them are dynamic variables, which are a magical Lisp feature you don't really see in other languages.

```clojure
(def ^:dynamic *max-rows* 100)

(defn x []
  (println *max-rows*)) 

(x) ; -> 100

(binding [*max-rows* 200]
  (x)) ; -> 200

(x) ; -> 100
```

In Clojure they're thread-local so millions of API calls can be going on at the same time and everyone will have the appropriate value of `*is-superuser?*` when the look at it